db_queries.php

```diff
@@ -49,6 +49,7 @@
                         <div class="col-md-3">
                             <div><input type="checkbox" id="type_forwarded" checked><label for="type_forwarded">Permitted: forwarded</label><br></div>
                             <div><input type="checkbox" id="type_cached" checked><label for="type_cached">Permitted: cached</label></div>
+                            <div><input type="checkbox" id="type_cached" checked><label for="type_cached_stale">Permitted: stale cache</label></div>
```

Comment:
You added 2 items with the same ID. You need to change the second one:
```suggestion
                            <div><input type="checkbox" id="type_cached" checked><label for="type_cached">Permitted: cached</label></div>
                            <div><input type="checkbox" id="type_cached_stale" checked><label for="type_cached_stale">Permitted: stale cache</label></div>
```